src/openrct2-ui/interface/InGameConsole.cpp

```diff
@@ -134,6 +134,12 @@ void InGameConsole::ScrollToEnd()
         _consoleScrollPos = std::max<int32_t>(0, static_cast<int32_t>(_consoleLines.size()) - maxLines);
 }
 
+void InGameConsole::RefreshCaret(size_t position)
+{
+    _consoleCaretTicks = 0;
+    _selectionStart = position;
+}
+
 void InGameConsole::RefreshCaret()
```

Comment:
You can eliminate this function by making the position parameter defaulting to 0.

Reply:
I did not know you could do this! I have got rid of the overloaded functions now.